pyindi/www/static/js/maps-indi.js

```diff
@@ -389,6 +407,40 @@ const updateProperties = (xml_text) => {
   return;
 }
 
+/**
+ * Function that scrapes the incoming xml for delProperty tags
+ * debugging is enabled.
+ * @param {XMLDocument} partial_doc XML text from INDI.
+ * @returns {XMLDocument} Copy of the partial document put in.
+ */
+const scrapeDeleteProperty = (partial_doc) => {
+  try {
+     var cp_partial_doc = partial_doc;
+
+    while ((match = XmlRegex.XML_DELPROPERTY.exec(partial_doc))) {
+      // Parse match
+      const parser = new DOMParser();
+      var dom = parser.parseFromString(match, "application/xml");
+      var root_node = dom.documentElement; // Root node
+
+
+      var device = root_node.getAttribute("device");
+      var name = root_node.getAttribute("name");
+      var timestamp = root_node.getAttribute("timestamp");
+
+      // Call the delete handler. 
+      updater.delete(device, name);
```

Comment:
A lot of the functions written use the `indi` payload built in `flattenIndiLess`. I think it would be beneficial to build a version of that. Then we can use some utility functions to get vector selectors. 

```suggestion
      var indi = {};
      indi.device = device;
      indi.name = name;
      indi.timestamp = timestamp;
      updater.delete(indi);
```
This does require us to modify the `updater.delete` method